src/package-url.d.ts

```diff
@@ -20,78 +20,159 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 */
 
-declare module "packageurl-js" {
+declare module "@jdalton/packageurl-js" {
```

Comment:
This should be changed I think to remove the `@jdalton`

Reply:
Yes, thank you! Fixed!